src/OpenTelemetry.Instrumentation.AspNetCore/OpenTelemetry.Instrumentation.AspNetCore.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

Comment:
Just curious, why was this needed?

Reply:
As we are using <FrameworkReference Include="Microsoft.AspNetCore.App" />, this is needed.
IIRC, if we do not add this, net5.0 targeted instrumentation library will be used for net6.0 app (should still work as we don't have any NET6 framework specific code).

Reply:
I don't _think_ it is needed until we want to use some piece of the .NET 6 API.

If the project is like this...

```csharp
<TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.1;net5.0</TargetFrameworks>

  <ItemGroup Condition="'$(TargetFramework)' == 'net5.0'">
    <FrameworkReference Include="Microsoft.AspNetCore.App" />
  </ItemGroup>
```

Users on .NET 6 will still be able to use it.

Reply:
Agreed - then do we remove 5.0 as well?. I have not tested but I think having only 3.1 will also work for 3.1+ apps then?.
But if we ever use version specific code, we will have to do add it again.
@cijothomas - thoughts?

Reply:
We don't use any .NET 6 specific feature today, but Asp.Net core 6.0 has introduced ActivitySource usage. If we want to leverage them, we either need .net6.0 target. or we use reflection to find if user is running Asp.Net Core 6.0 or newer.
(@vishweshbankwar please create a separate issue to discuss this further)